# translation of de.po to German
# translation of de.po to
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Mustafa Ölcerman <oelcerman@fussballkreis.org>, 2006.
# Mustafa Ölcerman <oelcerman@fussballkreis.org>, 2007.
# Mustafa Ölcerman <mustafa.oelcerman@fk-koeln.de>, 2007.
# Mustafa Ölcerman <m.oelcerman@pardus-linux.de>, 2007, 2008.
# Mustafa Ölcerman <m.oelcerman@pardus-linux.eu>, 2009.
# Klemens Häckel <click3d at linuxmail dot org>, 2009.
msgid ""
msgstr ""
"Project-Id-Version: de\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-03-18 15:24+0200\n"
"PO-Revision-Date: 2009-06-26 21:34-0400\n"
"Last-Translator: Klemens Häckel <click3d at linuxmail dot org>\n"
"Language-Team: German <es@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: KBabel 1.11.4\n"

#: bin/mudur.py:207
msgid "Cannot write mudur.log, read-only file system"
msgstr "Mudur.log kann nicht beschrieben werden, Dateisystem nur mit Lesezugriff"

#: bin/mudur.py:363
msgid "Safe Mode"
msgstr "Sicherer Modus"

#: bin/mudur.py:366
msgid "Cannot find /etc/pardus-release"
msgstr "Finde nicht /etc/pardus-release"

#: bin/mudur.py:458
msgid "/etc/env.d/03locale cannot be updated"
msgstr "/etc/env.d/03locale kann nicht aktualisiert werden"

#: bin/mudur.py:485
#, python-format
msgid "Could not set unicode mode on tty %d"
msgstr "Unicode konnte nicht an tty %d gesetzt werden"

#: bin/mudur.py:552
#, python-format
msgid "Bringing up %s"
msgstr "%s wird bereitgestellt"

#: bin/mudur.py:557
#, python-format
msgid "Unable to bring up %s"
msgstr "%s konnte nicht bereitgestellt werden"

#: bin/mudur.py:627
msgid "No network connection, skipping remote mount."
msgstr "Keine Netwerkverbindung vorhanden, Remote-Einbindung wird übersprungen."

#: bin/mudur.py:637
msgid "Cannot connect to DBus, services won't be started"
msgstr "Kann nicht mit DBus verbinden, Dienste werden nicht gestartet"

#: bin/mudur.py:658
#, python-format
msgid ""
"Unable to start network:\n"
"  %s"
msgstr ""
"Starten von Netzwerk fehlgeschlagen:\n"
"  %s"

#: bin/mudur.py:663
msgid "Cannot start system logger"
msgstr "System Logger kann nicht gestartet werden"

#: bin/mudur.py:666
msgid "Starting services"
msgstr "Dienste werden gestartet"

#: bin/mudur.py:698
msgid "Stopping services"
msgstr "Dienste werden angehalten"

#: bin/mudur.py:718 bin/mudur.py:738 bin/service.py:161 bin/service.py:270
#, python-format
msgid "Starting %s"
msgstr "%s wird gestartet"

#: bin/mudur.py:728 bin/mudur.py:745 bin/service.py:174 bin/service.py:282
#, python-format
msgid "Stopping %s"
msgstr "%s wird angehalten"

#: bin/mudur.py:770
msgid "Can't move persistent udev rules from /dev/.udev"
msgstr "Die beständigen udev-Regeln können nicht von /dev/.udev verschoben werden."

#: bin/mudur.py:776
msgid "Mounting /dev"
msgstr "/dev wird gemountet"

#: bin/mudur.py:818
msgid "Starting udev"
msgstr "udev wird gestartet"

#: bin/mudur.py:832
msgid "Populating /dev"
msgstr "/dev wird gefüllt"

#: bin/mudur.py:867
msgid "Removing stale lock file /etc/mtab~"
msgstr "Entferne die veraltete Sperrdatei /etc/mtab~"

#: bin/mudur.py:870
msgid "Failed removing stale lock file /etc/mtab~"
msgstr "Entfernen der Sperrdatei /etc/mtab~ fehlgeschlagen"

#: bin/mudur.py:881
msgid "/etc/fstab doesn't contain an entry for the root filesystem"
msgstr "/etc/fstab beinhaltet keinen Eintrag für das Root-Dateisystem"

#: bin/mudur.py:887
msgid "Remounting root filesystem read-only"
msgstr "Root-Dateisystem wird mit Lese-Zugriff erneut gemountet"

#: bin/mudur.py:892
msgid "Checking root filesystem (full check forced)"
msgstr "Root-Dateisystem wird überprüft (vollständige Überprüfung gezwungen)"

#: bin/mudur.py:898
msgid "Checking root filesystem"
msgstr "Root-Dateisystem wird überprüft"

#: bin/mudur.py:907
msgid "Filesystem repaired, but reboot needed!"
msgstr "Dateisystem wurde repariert, Neustart erforderlich!"

#: bin/mudur.py:911
msgid "Rebooting in 10 seconds..."
msgstr "Computer wird in 10 Sekunden neu gestartet"

#: bin/mudur.py:913
msgid "Rebooting..."
msgstr "Wird neu gestartet..."

#: bin/mudur.py:917
msgid "Filesystem could not be repaired"
msgstr "Dateisystem konnte nicht fixiert werden"

#: bin/mudur.py:922
msgid "Skipping root filesystem check (fstab's passno == 0)"
msgstr "Root Dateisystem Überprüfung (fstab's passno == 0) wird übersprungen"

#: bin/mudur.py:928
msgid "Remounting root filesystem read/write"
msgstr "Root Dateisystem wird neu mit Lese-/Schreibzugriff gemountet"

#: bin/mudur.py:932
msgid ""
"Root filesystem could not be mounted read/write\n"
"   You can either login below and manually check your filesytem(s) OR\n"
"   restart your system, press F3 and select 'FS check' from boot menu\n"
msgstr ""
"Root Dateisystem konnte nicht für Lese-/Schreibzugriff gemounted werden.\n"
"   Loggen Sie sich entweder hier unterhalb ein und prüfen Sie manuell das Dateisystem, ODER\n"
"   starten Sie das System neu, drücken F3 und wählen 'FS check' aus dem Boot-Menü aus.\n"

#: bin/mudur.py:944
msgid "Couldn't synchronize /etc/mtab from /proc/mounts"
msgstr "Konnte /etc/mtab nicht von /proc/mounts synchronisieren"

#: bin/mudur.py:963
msgid "Checking all filesystems"
msgstr "Alle Dateisysteme werden überprüft"

#: bin/mudur.py:967
msgid "A full fsck has been forced"
msgstr "Ein vollständiges fsck wurde erzwungen"

#: bin/mudur.py:985
msgid "Filesystem errors corrected"
msgstr "Fehler im Dateisystem korrigiert"

#: bin/mudur.py:987
msgid "Fsck could not correct all errors, manual repair needed"
msgstr "Fsck konnte nicht alle Fehler beheben, eine manuelle Behebung ist nötig."

#: bin/mudur.py:995
msgid "Mounting USB filesystem"
msgstr "USB-Dateisystem wird gemountet"

#: bin/mudur.py:998
msgid "Mounting local filesystems"
msgstr "Lokale Dateisysteme werden gemountet"

#: bin/mudur.py:1018
msgid "Starting portmap service for NFS"
msgstr "Portmap-Dienst für NFS wird gestartet"

#: bin/mudur.py:1021
msgid "Mounting remote filesystems (CTRL-C stops trying)"
msgstr "Remote-Dateisysteme werden gemountet (STRG-C stoppt den Versuch)"

#: bin/mudur.py:1035
msgid "Mounting skipped with CTRL-C, remote shares will not be accessible!"
msgstr "Mounten mit STRG-C übersprungen, Remote-Freigaben sind nicht zugänglich!"

#: bin/mudur.py:1078
#, python-format
msgid "Setting up hostname as '%s'"
msgstr "Hostname wird als '%s' eingestellt"

#: bin/mudur.py:1103
msgid "Setting disk parameters"
msgstr "Festplatten-Parameter werden gesetzt"

#: bin/mudur.py:1125
msgid "Activating swap space"
msgstr "SWAP wird aktiviert"

#: bin/mudur.py:1134
msgid "Deactivating swap space"
msgstr "SWAP wird deaktiviert"

#: bin/mudur.py:1143
msgid "Cleaning up /var"
msgstr "/var wird gesäubert"

#: bin/mudur.py:1154
msgid "Cleaning up /tmp"
msgstr "/tmp wird gesäubert"

#: bin/mudur.py:1189
msgid "Setting system clock to hardware clock"
msgstr "Systemuhr wird von der Hardwareuhr gesetzt"

#: bin/mudur.py:1205
msgid "Failed to adjust systematic drift of the hardware clock"
msgstr "Abgleichung der systematischen Abweichung von der Hardware-Uhr ist fehlgeschlagen"

#: bin/mudur.py:1209
msgid "Failed to set system clock to hardware clock"
msgstr "Einstellen der Systemuhr von der Hardwareuhr ist fehlgeschlagen"

#: bin/mudur.py:1220
msgid "Syncing system clock to hardware clock"
msgstr "Systemuhr wird mit der Hardwareuhr synchronisiert"

#: bin/mudur.py:1223
msgid "Failed to synchronize clocks"
msgstr "Synchronisation der Uhr ist fehlgeschlagen"

#: bin/mudur.py:1250
msgid "Unmounting filesystems"
msgstr "Unmounten der Dateisysteme"

#: bin/mudur.py:1282
msgid "Remounting remaining filesystems read-only"
msgstr "Restliche Dateisysteme werden mit Lesezugriff erneut gemountet"

#: bin/mudur.py:1298
msgid "An internal error occured. Please report to the bugs.pardus.org.tr with following information:"
msgstr "Ein Interner Fehler ist vorgekommen. Bitte an bugs.pardus.org.tr mit der folgenden Information melden:"

#: bin/mudur.py:1359
msgid "Mounting /sys"
msgstr "/sys wird gemountet"

#: bin/mudur.py:1373
msgid "Mounting /dev/pts"
msgstr "/dev/pts wird gemountet"

#: bin/mudur.py:1422
msgid "Setting up localhost"
msgstr "Localhost wird eingestellt"

#: bin/mudur.py:1435
msgid "Updating environment variables"
msgstr "Umgebungsvariablen werden aktualisiert"

#: bin/mudur.py:1456
msgid "Triggering udev events which are failed during a previous run"
msgstr "Udev-Ereignisse, die bei der vorherigen Ausführung fehlgeschlagen sind, werden angesteuert"

#: bin/mudur.py:1495
msgid "Trying to initiate a warm reboot (skipping BIOS with kexec kernel)"
msgstr "Wird versucht, ein Warmstart zu beginnen (überspringe BIOS mit kexec kernel)"

#: bin/service.py:60
msgid "local"
msgstr "Lokal"

#: bin/service.py:61
msgid "script"
msgstr "Skript"

#: bin/service.py:62
msgid "server"
msgstr "Server"

#: bin/service.py:74
msgid "running"
msgstr "wird ausgeführt"

#: bin/service.py:76
msgid "yes"
msgstr "ja"

#: bin/service.py:78
msgid "conditional"
msgstr "abhängig"

#: bin/service.py:101
msgid "Status"
msgstr "Status"

#: bin/service.py:102
msgid "Service"
msgstr "Dienst"

#: bin/service.py:103
msgid "Autostart"
msgstr "Autostart"

#: bin/service.py:104
msgid "Description"
msgstr "Beschreibung"

#: bin/service.py:147 bin/service.py:157
#, python-format
msgid "Unable to start %s:"
msgstr "Konnte %s nicht starten:"

#: bin/service.py:170
#, python-format
msgid "Unable to stop %s:"
msgstr "Konnte %s nicht stoppen:"

#: bin/service.py:183
#, python-format
msgid "Unable to set %s state:"
msgstr "Konnte %s Status nicht setzen:"

#: bin/service.py:188
#, python-format
msgid "Service '%s' will be auto started."
msgstr "Dienst '%s' wird automatisch gestartet."

#: bin/service.py:190
#, python-format
msgid "Service '%s' won't be auto started."
msgstr "Dienst '%s' wird nicht automatisch gestartet."

#: bin/service.py:192
#, python-format
msgid "Service '%s' will be started if required."
msgstr "Dienst '%s' wird bei Bedarf gestartet."

#: bin/service.py:201
#, python-format
msgid "Unable to reload %s:"
msgstr "%s konnte nicht bereitgestellt werden"

#: bin/service.py:205
#, python-format
msgid "Reloading %s"
msgstr "%s wird neu geladen"

#: bin/service.py:259
msgid "You must be root to use that."
msgstr "Sie müssen als Root angemeldet sein, um dies zu verwenden."

#: bin/service.py:278
msgid "Unable to start DBus"
msgstr "DBus konnte nicht gestartet werden"

#: bin/service.py:292
msgid "DBus is not running."
msgstr "DBus läuft nicht."

#: bin/service.py:294
msgid "DBus is running."
msgstr "DBus wird ausgeführt."

#: bin/service.py:299
msgid ""
"usage: service [<options>] [<service>] <command>\n"
"where command is:\n"
" list     Display service list\n"
" status   Display service status\n"
" info     Display service status\n"
" on       Auto start the service\n"
" off      Don't auto start the service\n"
" start    Start the service\n"
" stop     Stop the service\n"
" restart  Stop the service, then start again\n"
" reload   Reload the configuration (if service supports this)\n"
"and option is:\n"
" -N, --no-color  Don't use color in output\n"
" -q, --quiet     Don't print replies"
msgstr ""
"Verwendung: service [<Option>][<Dienst>] <Befehl>\n"
"wo Befehl folgendes ist:\n"
" list    Dienste auflisten \n"
" status   Status des Dienstes anzeigen\n"
" info    Status des Dienstes anzeigen\n"
" on      Dienst automatisch starten\n"
" off      Dienst nicht automatisch starten\n"
" start   Dienst starten\n"
" stop   Dienst anhalten\n"
" restart Dienst anhalten und anschliessen erneut starten\n"
" reload Konfiguration wird neugeladen (wenn der Dienst das unterstützt)\n"
"und Option ist:\n"
" -N, --no-color  keine Farben bei der Ausgabe verwenden\n"
" -q, --quiet     Keine Antworten ausgeben"

#: bin/service.py:355
#, python-format
msgid "No such service: %s"
msgstr "Kein solcher Service: %s"

#~ msgid "Root filesystem could not be mounted read/write"
#~ msgstr "Root Dateisystem konnte nicht mit Lese-/Schreibzugriff gemountet werden"
